test/AngularSpec.js

```diff
@@ -228,6 +228,18 @@ describe('angular', function() {
       }
     });
 
+    it('should handle Blob objects', function() {
+      if (typeof Blob !== 'undefined') {
+        var src = new Blob(['foo'], {type: 'bar'});
+        var dst = copy(src);
+
+        expect(dst).not.toBe(src);
+        expect(dst.size).toBe(3);
+        expect(dst.type).toBe('bar');
+        expect(isBlob(dst)).toBe(true);
```

Comment:
BTW, I don't think `typeof` returns `Blob` for blobs. In latest Chrome and Firefox at least, it return `object`.
(Maybe what you had in mind is that `expect(typeof dst).toBe(typeof src)` would indeed pass even without the fix (because both would return `object`).)